### PR TITLE
Skip https certificate verification

### PR DESCRIPTION
**Pull request by fengshunli**
This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### CODE REVIEW

File: cmd/polaris/audit.go [hunk 1]
```diff
@@ -202,7 +203,9 @@ func outputAudit(auditData validator.AuditData, outputFile, outputURL, outputFor
 			} else {
 				req.Header.Set("Content-Type", "text/plain")
 			}
-			client := &http.Client{}
+			client := &http.Client{Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+			}}
```
Comment (rbren) [inline cmd/polaris/audit.go:208]: @fengshunli this is super dangerous--at most we should make this an option and turn it off by default.

**Comment by rbren** [comment]
@fengshunli can you fill out the PR template? I'm not sure what you're trying to achieve here.

It might be better to provide a CA using these environment variables
```
SSL_CERT_FILE=/path/to/x509_encoded_cert_file dep ensure
SSL_CERT_DIR=/path/to/dir/ dep ensure
```

**Comment by fengshunli** [comment]
Because I am a self-signed certificate, I want to skip the certificate verification when uploading the result through --output-url @rbren

**Comment by fengshunli** [comment]
Do I need to add a global -skip-ssl-validation set to false by default to resubmit @rbren

**Comment by rbren** [comment]
Yup that's perfect! Thanks

**Comment by fengshunli** [comment]
> Yup that's perfect! Thanks

done @rbren

File: cmd/polaris/audit.go [hunk 2]
```diff
@@ -202,7 +205,12 @@ func outputAudit(auditData validator.AuditData, outputFile, outputURL, outputFor
 			} else {
 				req.Header.Set("Content-Type", "text/plain")
 			}
-			client := &http.Client{}
+
+			transport := &http.Transport{}
+			if skipSslValidation {
+				transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+			}
+			client := &http.Client{Transport: transport}
```
Comment (rbren) [inline cmd/polaris/audit.go:213]:
```suggestion
			client := &http.Client{}
			if skipSslValidation {
				transport := &http.Transport{}
				transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
				client = &http.Client{Transport: transport}
			}
```

**Comment by fengshunli** [comment]
done @rbren

**Review by rbren — APPROVED** [review]

**Pull request merged**